test/fiddle/test_closure.rb

```diff
@@ -87,6 +87,18 @@ def call(bool)
       end
     end
 
+    def test_pointer
+      closure_class = Class.new(Closure) do
+        def call(ptr)
+          ptr.is_a?(Pointer)
+        end
+      end
+      closure_class.create(:bool, [:voidp]) do |closure|
+        func = Function.new(closure, [:voidp], :bool)
+        assert_equal(true, func.call(Pointer["hello"]))
```

Comment:
Could you use power assert style for boolean?

```suggestion
        assert do
          func.call(Pointer["hello"])
        end
```